metrics/grafana/tikv_details.json

```diff
@@ -16810,6 +16817,109 @@
             "alignLevel": null
           }
         },
+        {
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "datasource": "${DS_TEST-CLUSTER}",
+          "dashes": false,
+          "description": "The number of pending destroy region task",
+          "editable": true,
+          "error": false,
+          "fieldConfig": {
+            "defaults": {},
+            "overrides": []
+          },
+          "fill": 1,
+          "fillGradient": 0,
+          "grid": {},
+          "gridPos": {
+            "h": 7,
+            "w": 8,
+            "x": 16,
+            "y": 61
+          },
+          "hiddenSeries": false,
+          "id": 12308,
+          "legend": {
+            "alignAsTable": true,
+            "avg": false,
+            "current": true,
+            "hideEmpty": false,
+            "max": true,
+            "min": false,
+            "rightSide": true,
+            "show": true,
+            "sort": "current",
+            "sortDesc": true,
+            "total": false,
+            "values": true
+          },
+          "lines": true,
+          "linewidth": 1,
+          "links": [],
+          "nullPointMode": "null",
+          "percentage": false,
+          "pointradius": 5,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": true,
+          "targets": [
+            {
+              "expr": "sum(tikv_pending_delete_ranges_of_stale_peer{tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}) by (instance)",
```

Comment:
Can you post an image in description to show how it looks?

Reply:
![image](https://user-images.githubusercontent.com/16376959/130012612-3a36f94a-c6ce-4616-8982-4c2466b6e0c1.png)
![image](https://user-images.githubusercontent.com/16376959/130012702-ef08063a-a5bd-4bf5-a2d1-50fdf2dee879.png)